x-pack/plugins/observability_solution/inventory/public/routes/config.tsx

```diff
@@ -28,19 +27,18 @@ const inventoryRoutes = {
         t.type({
           sortField: entityColumnIdsRt,
           sortDirection: t.union([t.literal('asc'), t.literal('desc')]),
-          pageIndex: toNumberRt,
         }),
         t.partial({
           entityTypes: entityTypesRt,
           kuery: t.string,
+          view: t.string,
```

Comment:
Have you thought about having a param to control the page view: `unifed` | `grouping` and another to control the groupBy fields, this one could even be an array? This way you don't have to use an object, which will simply the implementation.

Reply:
I'm passing on making this an array for now, because there's no need to add complexity to the PR when we are only doing a single level of grouping and not multiple. In any case, the view type and pagination states have been separated for now.